src/main/java/xyz/nucleoid/spleef/game/SpleefTimerBar.java

```diff
@@ -28,7 +28,7 @@ public void update(long ticksUntilDrop, long totalTicksUntilDrop) {
     }
 
     public void setBarLava(){
-        var gameName = new TranslatableText("game.spleef.spleef").formatted(Formatting.BOLD);
+        var gameName = new TranslatableText("gameType.spleef.spleef").formatted(Formatting.BOLD);
         var lavaMsg = new TranslatableText("game.spleef.lava.msg");
         this.widget.setTitle(new LiteralText("").append(gameName).append(" - ").append(lavaMsg).formatted(Formatting.RED));
```

Comment:
It would be better to reuse the existing `getBarTitle` helper, as the only difference is the formatting of the text.

Reply:
I tried this but ran into issues formatting the text to be all red as the `formatted` method is not included in `getBarTitle`. What's the best way to resolve this? I came up with two ideas, make a new helper method that is a copy of `getBarTitle` but the text is formatted red or use a method that converts the output of `getBarTitle` to LiteralText so it can be formatted.

Reply:
It's fine to add a parameter to replace `Formatting.GREEN`.